SimBatchDelete/ViewController.swift

```diff
@@ -211,5 +191,57 @@ extension ViewController: NSTableViewDelegate {
 
         return cell
     }
+    
+    override func keyDown(with event: NSEvent) {
+        guard event.keyCode == 51 else {
```

Comment:
Shouldn't there be a constant for the key instead of the plain number?
Also, I don't see a check for the `command` modifier.

Reply:
Please check this: https://stackoverflow.com/a/27621180/2252279
for `NSBackspaceCharacter` and event.modifierFlags.contains(.command)`

Reply:
make sense will to the required changes, tonight Thanks :)

Reply:
Also regarding the Command key check, it will always be in `func keyup` if only delete is pressed. I guess that will do the thing